Symfony/CS/Fixer/Symfony/PhpdocAnnotationWithoutDotFixer.php

```diff
@@ -43,7 +43,12 @@ public function fix(\SplFileInfo $file, $content)
             foreach ($annotations as $annotation) {
                 if ($annotation->getTag()->valid()) {
                     $line = $doc->getLine($annotation->getEnd());
-                    $line->setContent(preg_replace('/[.。](\s+)$/u', '\1', $line->getContent()));
+
+                    preg_match('/^(.*[.。])(\s*)$/', $line->getContent(), $matches);
+
+                    if (0 !== count($matches)) {
+                        $line->setContent(mb_substr($matches[1], 0, -1).$matches[2]);
```

Comment:
the `mb` package is not installed on a default PHP install, so we should at it to `composer.json` as req.?